src/test/java/com/google/sps/data/DrinkDAOTest.java

```diff
@@ -96,26 +115,31 @@ public void testSearchForDrinkByRating() {
 
   @Test
   public void testSearchForDrink() {
-    Drink a = DrinkDAO.saveDrink(DRINK_A, ratingA, numRatingsA, storeIDA);
-    Drink b = DrinkDAO.saveDrink(DRINK_A, ratingB, numRatingsB, storeIDB);
-    Drink c = DrinkDAO.saveDrink(DRINK_B, ratingC, numRatingsC, storeIDC);
-    Drink d = DrinkDAO.saveDrink(DRINK_B, ratingD, numRatingsD, storeIDD);
+    Drink a = DrinkDAO.saveDrink(DRINK_A, ratingA, storeIDA);
+    Drink b = DrinkDAO.saveDrink(DRINK_A, ratingB, storeIDB);
+    Drink c = DrinkDAO.saveDrink(DRINK_B, ratingC, storeIDC);
+    Drink d = DrinkDAO.saveDrink(DRINK_B, ratingD, storeIDD);
 
     Set<Drink> drinks = Drink.searchForDrink(DRINK_A);
     for (Drink drink : drinks) {
+      Assert.assertTrue(drink.getName().equals(DRINK_A));
       Assert.assertTrue((drink.getStore().equals(storeIDA)) || (drink.getStore().equals(storeIDB)));
     }
     drinks = Drink.searchForDrink(DRINK_B);
     for (Drink drink : drinks) {
+      Assert.assertTrue(drink.getName().equals(DRINK_B));
       Assert.assertTrue((drink.getStore().equals(storeIDC)) || (drink.getStore().equals(storeIDD)));
```

Comment:
ditto

Reply:
same here: assert that "drinks" is of size 2 and contains both storeC and storeD.